scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -82,7 +82,7 @@ class FossId(
 
     companion object {
         @JvmStatic
-        private val PROJECT_NAME_REGEX = Regex("""^.*\/([\w-]+)\.git$""")
+        private val PROJECT_NAME_REGEX = Regex("""^.*\/([\w\-]+)\.git$""")
```

Comment:
AFAIK chars inside `[]` of a Regex are literal, i.e. the escaping should not be needed.

Reply:
`-` is used to specify character range inside `[]`, e.g. `[a-z]`

Reply:
Ah, right, thanks for the reminder.

Reply:
More commit message fixes for that commit:
- s/accept/Accept/
- s/urls/URLs/